src/openrct2/rct2/S6Importer.cpp

```diff
@@ -544,18 +544,73 @@ namespace RCT2
         {
             if (String::Equals(_s6.scenario_filename, "Europe - European Cultural Festival.SC6"))
             {
-                // This scenario breaks pathfinding. Create passages between the worlds. (List is grouped by neighbouring
-                // tiles.)
-                // clang-format off
+                // This scenario breaks pathfinding
                 FixLandOwnershipTilesWithOwnership(
                     {
-                        { 67, 94 }, { 68, 94 }, { 69, 94 },
-                        { 58, 24 }, { 58, 25 }, { 58, 26 }, { 58, 27 }, { 58, 28 }, { 58, 29 }, { 58, 30 }, { 58, 31 }, { 58, 32 },
-                        { 26, 44 }, { 26, 45 },
-                        { 32, 79 }, { 32, 80 }, { 32, 81 },
+                        { 67, 94 },
+                        { 68, 94 },
+                        { 69, 94 },
+                        { 58, 24 },
+                        { 58, 25 },
+                        { 58, 26 },
+                        { 58, 27 },
+                        { 58, 28 },
+                        { 58, 29 },
+                        { 58, 30 },
+                        { 58, 31 },
+                        { 58, 32 },
+                        { 26, 44 },
+                        { 26, 45 },
+                        { 32, 79 },
+                        { 32, 80 },
+                        { 32, 81 },
+                    },
+                    OWNERSHIP_OWNED);
+            }
+            else if (String::Equals(gScenarioFileName, "Six Flags Holland.SC6"))
+            {
+                FixLandOwnershipTilesWithOwnership(
+                    {
+                        { 112, 33 },
+                        { 112, 34 },
+                        { 113, 117 },
+                        { 114, 117 },
+                        { 115, 117 },
+                        { 116, 117 },
+                        { 117, 117 },
+                        { 114, 118 },
+                        { 115, 118 },
+                        { 116, 118 },
+                        { 117, 118 },
+                    },
+                    OWNERSHIP_AVAILABLE, true);
+            }
+            else if (String::Equals(gScenarioFileName, "North America - Grand Canyon.SC6"))
+            {
+                FixLandOwnershipTilesWithOwnership(
+                    {
+                        { 131, 90 }, { 132, 90 }, { 133, 91 }, { 134, 91 }, { 135, 91 }, { 136, 91 },
+                        // second group missing from reddit below (placeholder comment)
```

Comment:
What does this mean?

Reply:
it means there are more tiles in the scenario that need to be added

Reply:
I asked gymnasiast to finish this PR because I discovered in the middle of coding it, he was self assigned, thats why I have draft in the title because I simply added my part for gymnasiast to look at to save him time when he gets around to the rest of the code, so whatever changes are necessary, I'm trying not to implement myself